tests/cbor_decode/test5_strange/src/main.c

```diff
@@ -91,7 +93,7 @@ void test_strings(void)
 		0,1,2,3,4,5,6,7,8,9,0,1,2,3,4,5,6,7,8,9,0,1,2,3,4,5,6,7,8,9,
 		0xC0, 0x78, 0x1E, // 30 bytes
 		0,1,2,3,4,5,6,7,8,9,0,1,2,3,4,5,6,7,8,9,0,1,2,3,4,5,6,7,8,9,
-		0x58, 29, // Numbers (len: 29)
+		0x58, STR_LEN(29, 1), // Numbers (len: 29)
```

Comment:
Are the changes to this file supposed to be part of this commit/PR?

Reply:
No, they are a fix to the indeterminate length array PR. I put it in a separate commit now.